xoto3/dynamodb/update/versioned.py

```diff
@@ -163,8 +167,11 @@ def versioned_diffed_update_item(
             else:
                 raise
     raise VersionedUpdateFailure(
```

Comment:
Should this also be given the magical name treatment?

Reply:
yeah, maybe so. I already can't remember whether i considered it and rejected it or just didn't get that far. i will go back over this and see if i can put myself back in that mindset.

Reply:
oh, i remember now. `versioned_diffed_update_item` does not (yet) accept a nicename parameter, and I had decided against doing anything like dynamically using the table name. 

I _could_ introduce a nicename parameter, but I had mostly decided that since nothing would be taking advantage of it immediately (i.e. just by upgrading), it was less compelling.

Still, I'm not opposed.

Reply:
i have done this. could be nice to have in various places now that it's there for the using.